pages/monitoring.md

```diff
@@ -48,6 +49,17 @@ The health dashboard uses Spring Boot Actuator's health endpoint to give health
 The logs dashboard allows to manage at runtime the Logback configuration of the running application. 
 You can change the log level of a Java package by clicking on a button, which is very convenient both in development and in production.
 
+<h2 id="security-metrics">Security Metrics</h2>
```

Comment:
You don't need to wrap this in an H2 if the id matches what would be generated. You can just do:

```
## Security Metrics
```

Reply:
Updated! Please, be aware that there are other occurences in the same file.

Reply:
Shall I fix those too?

Reply:
Sure!

Reply:
Fixed.
I've also removed this items, since it doesn't point to any existing sub-chapter: 

![image](https://user-images.githubusercontent.com/32668766/140616556-db378e9c-92b0-42c4-9d35-297abe5fd3e3.png)